WEB_SERVICE/WEB/_HIGH/LDAP_INJECTION/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "risk_rating": "high",
+  "short_description": "The web application is vulnerable to an LDAP injection vulnerability.",
+  "references": {
+    "OWASP": "https://owasp.org/www-community/attacks/LDAP_Injection",
+    "OWASP cheatsheets":"https://cheatsheetseries.owasp.org/cheatsheets/LDAP_Injection_Prevention_Cheat_Sheet.html",
+    "Hacktricks": "https://book.hacktricks.xyz/pentesting-web/ldap-injection"
+  },
+  "title": "LDAP Injection",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "CWE_TOP_25": [
+      "CWE_90"
+    ],
+    "GDPR": [
+      "ART_25",
+      "ART_32"
+    ],
+    "PCI_STANDARDS":[
```

Comment:
I think "REQ_6_2" and "REQ_6_3" should be added too !! 
![image](https://github.com/Ostorlab/KB/assets/144013278/712807e2-2f6c-44e6-b2c9-c0f879def89d)